### DISCUSSION TITLE
using of @babel/parse with babylon-walk

### DISCUSSION THREAD

**Discussion by chaitanya71998** [babel/babel#12747] [Q&A]
can we use @babel/parse with babylon-walk, will there be any issues or should we only use babel/traverse

**Comment by JLHwung** [comment]
It has been renamed as `babel-walk` since then which supports Babel 7: https://github.com/pugjs/babel-walk, as you have raised an issue there.

`@babel/parse` should work with any worker as long as the worker respects the AST shape. We will not forbid using `babel-walk` but it is not maintained by the Babel team. If you have any issue on using `babel-walk`, please file a report there.
